development/Dockerfile

```diff
@@ -1,44 +1,71 @@
-ARG NAUTOBOT_VER="1.4.0"
-ARG PYTHON_VER=3.7
+# -------------------------------------------------------------------------------------
+# Nautobot Plugin Developement Dockerfile Template
+# Version: 0.0.1
+#
+# Plugins that need to add additional steps or packages can do in the section below.
+# -------------------------------------------------------------------------------------
+# !!! CAUTION MODIFYING LINES BELOW
 
-FROM ghcr.io/nautobot/nautobot-dev:${NAUTOBOT_VER}-py${PYTHON_VER}
+# Accepts a desired Nautobot version as build argument, default to 1.2.0
+ARG NAUTOBOT_VER="1.2.0"
```

Comment:
For this project in its current state it looks like 1.4.0 was the previous default?